Boardfiles/zeropilot-3-0/.settings/language.settings.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I don't think we need the .settings folder to be updated, but maybe this is ok?

Reply:
maybe this is fine? This is the initial commit to this board file, I guess there gotta be some setting exists. I am not sure if people are gonna like my setting tho

Reply:
Initial commit I think it's fine,

Once it exists I don't think we need updates. It can screw with a lot of files .... but hopefully all is good!